.github/workflows/ovn-ovs-coverage.yml

```diff
@@ -0,0 +1,46 @@
+name: OVN and OVS coverage
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Run workflows weekly, 02:21 every Tuesday
+    - cron: "21 2 * * 3"
+
+jobs:
+  coverage:
+    runs-on: [self-hosted, amd64, tiobe, noble]
+    strategy:
+      matrix:
```

Comment:
By adding a [`fail-fast`](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#jobsjob_idstrategyfail-fast) key here, we could have results on one of the job even if the other fail

Reply:
@mkalcok adding fail-fast would likely also avoid cancelling the running job in the event of the other failing, which I think might attribute to the server side lock?